src/Components/WebAssembly/Build/src/targets/ServiceWorkerAssetsManifest.targets

```diff
@@ -89,7 +89,7 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="_OmitServiceWorkerContent" BeforeTargets="AssignTargetPaths">
+  <Target Name="_OmitServiceWorkerContent" BeforeTargets="AssignTargetPaths; ResolveCurrentProjectStaticWebAssetsInputs">
```

Comment:
This should work with just "ResolveCurrentProjectStaticWebAssetsInputs" does it not?

Reply:
No. Sometimes `AssignTargetPaths` runs much earlier than `ResolveCurrentProjectStaticWebAssetsInputs`, and in that case unless we already ran this code, we would end up with unwanted `<ContentWithTargetPath>` entries causing the `service-worker.published.js` file to be emitted to the publish output.